.ci/prepare-git-context.sh

```diff
@@ -25,3 +19,9 @@ git checkout master
 # when running the release pipeline.
 # used GIT_BASE_COMMIT instead GIT_COMMIT to support the MultiBranchPipelines.
 git reset --hard "${GIT_BASE_COMMIT}"
+
+# Force the git user details when pushing using the GIT_BASE_COMMIT commit details
+USER_MAIL=$(git log -1 --pretty=format:'%ae')
+USER_NAME=$(git log -1 --pretty=format:'%an')
+git config --global user.email "${USER_MAIL}"
```

Comment:
I'd preferred to touch only the current repo config (not global), https://help.github.com/en/github/using-git/setting-your-username-in-git#setting-your-git-username-for-a-single-repository

```
git config user.email "${USER_MAIL}"
git config user.name "${USER_NAME}"
```

Reply:
It's within the context for the ephemeral worker and docker container, so I guess no issues in the CI, but likely something to be changed to avoid major issues when running the script locally :)